contracts/DFRewards.sol

```diff
@@ -1,20 +1,19 @@
-pragma solidity 0.8.12;
 // SPDX-License-Identifier: (Apache-2.0 AND CC-BY-4.0)
+pragma solidity 0.8.12;
 
 import "OpenZeppelin/openzeppelin-contracts@4.2.0/contracts/token/ERC20/IERC20.sol";
 import "OpenZeppelin/openzeppelin-contracts@4.2.0/contracts/token/ERC20/utils/SafeERC20.sol";
 import "OpenZeppelin/openzeppelin-contracts@4.2.0/contracts/security/ReentrancyGuard.sol";
 import "OpenZeppelin/openzeppelin-contracts@4.2.0/contracts/access/Ownable.sol";
+import "./interfaces/IDFRewards.sol";
 
-contract DFRewards is Ownable, ReentrancyGuard {
+contract DFRewards is Ownable, ReentrancyGuard, IDFRewards {
     using SafeERC20 for IERC20;
 
-    event Allocated(address[] tos, uint256[] values, address tokenAddress);
-    event Claimed(address to, uint256 value, address tokenAddress);
-
     // token address => user address => balance
     mapping(address => mapping(address => uint256)) balances;
     mapping(address => uint256) allocated;
+    mapping(address => bool) strategies;
```

Comment:
Perhaps 'live_strategies' is a better label than 'strategies'?

And, should it be simply a _list_ of live strategies, rather than a boolean mapping?

Reply:
Re live_strategies - I see you changed it, great!